geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/Z3IdxStrategy.scala

```diff
@@ -13,11 +13,10 @@ import com.vividsolutions.jts.geom.{Geometry, GeometryCollection}
 import org.apache.accumulo.core.data.Range
```

Comment:
I added a ticket [here](https://geomesa.atlassian.net/browse/GEOMESA-1014) to update the docs/comments on query planning.